apps/betterangels-backend/accounts/managers.py

```diff
@@ -9,25 +9,24 @@
 
 
 class UserManager(BaseUserManager["User"]):
-    def create_user(self, email: str, password: str = "", **extra_fields: Any) -> "User":
```

Comment:
I would prefer if we are passing optional args that we explicitly pass them as optional and not rely on extra fields.

Reply:
realized all these methods weren't necessary since we were just using the wrong base class and then just reimplementing the same functionality that django has for free in `from django.contrib.auth.models import UserManager as DjangoUserManager`